### PR TITLE
Add cases and deaths by age, race, ethnicity, and sex to Tennessee scraper

### PR DESCRIPTION
**Pull request by jrybacek**
Add data not currently scraped from official TN website files https://www.tn.gov/health/cedep/ncov/data/downloadable-datasets.html. This scraper targets the `Age by County` spreadsheet https://www.tn.gov/content/dam/tn/health/documents/cedep/novel-coronavirus/datasets/Public-Dataset-Daily-County-Age-Group.XLSX and the `Race, Ethnicity, Sex` spreadsheet https://www.tn.gov/content/dam/tn/health/documents/cedep/novel-coronavirus/datasets/Public-Dataset-RaceEthSex.XLSX.

Minor changes:
- Add a required library to the requirements-dev.txt file, so other developers don't have issues when on-boarding
- Remove irrelevant database setup information in `README.md`

### CODE REVIEW

**Comment by jrybacek** [comment]
My apologies tests on my local environment using `pytest -v -k Tennessee` work without failing.  I'm not sure what's causing this error.

**Comment by jrybacek** [comment]
> My apologies tests on my local environment using `pytest -v -k Tennessee` work without failing. I'm not sure what's causing this error.

This issue is now fixed, please merge when once we've reviewed the results.

File: README.md [hunk 1]
```diff
@@ -11,48 +11,6 @@
...
-df = scraper.normalize(scraper.fetch())
-
-scraper.put(engine, df)
-```
-
-Note that by default the `create_dev_engine` routine will construct the database
-in a verbose mode where all SQL commands are echoed to the console. We find that
-this is helpful while debugging and developing. This can be disabled by passing
-`verbose=False` when calling `create_dev_engine`.
-
```
Comment (sglyon) [inline README.md:55]: Hey @jrybacek was removing this section of the docs intentional?

I believe it is still accurate. If not, please let me know.

File: can_tools/scrapers/official/TN/tn_state.py [hunk 2]
```diff
@@ -292,3 +292,223 @@ def normalize(self, data) -> pd.DataFrame:
...
+        df.loc[df["category_detail"] == "White", "category_detail"] = "white"
+        df.loc[
+            df["category_detail"] == "Native Hawaiian or Other Pacific Islander",
+            "category_detail",
+        ] = "pacific_islander"
+        df.loc[
+            df["category_detail"] == "Other/ Multiracial", "category_detail"
+        ] = "multiple_other"
+        df.loc[
+            df["category_detail"] == "Other/Multiracial", "category_detail"
```
Comment (sglyon) [inline can_tools/scrapers/official/TN/tn_state.py:442]: ahh how irritating that they have two versions of this!

Good catch!

**Review by sglyon — COMMENTED** [review]
Great work -- only remaining comment is about the deletion of the sqlalchemy content from the readme

**Review by cc7768 — APPROVED** [review]
This is great! Thanks for taking this on -- I made a few small changes but otherwise looks good and ready to merge.

File: README.md (hunk 1 above)
Comment (jrybacek) [reply README.md:55]: Hmm, yeah.  Its still accurate.  At the time I read it, I was thinking it was extraneous, but it does make sense to keep for anyone hoping to run a local PostgreSQL server.  I'll revert this change.

**Pull request merged**